test/functional/apps/home/_newsfeed.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ getService, getPageObjects }: FtrProviderContext) {
+  const globalNav = getService('globalNav');
+  const PageObjects = getPageObjects(['common', 'newsfeed']);
+
+  describe('Newsfeed icon button', () => {
```

Comment:
Feels like the description of this `describe` is only correctly for two of these tests, but not the others.
Perhaps we should split it into two `describe`s?

Reply:
I think the description is just accidentally specific to 1 thing. It makes sense to me to keep it as 1 describe though. The only thing we'd gain from split-up is longer test run: we'd have to change `before` to `beforeEach`